pkg/trafficpolicy/trafficpolicy.go

```diff
@@ -10,10 +10,15 @@ import (
 )
 
 // NewRouteWeightedCluster takes a route and weighted cluster and returns a *RouteWeightedCluster
-func NewRouteWeightedCluster(route HTTPRouteMatch, weightedCluster service.WeightedCluster) *RouteWeightedClusters {
+func NewRouteWeightedCluster(route HTTPRouteMatch, weightedClusters []service.WeightedCluster) *RouteWeightedClusters {
+	weightedClusterSet := set.NewSet()
```

Comment:
I wish we used `set` instead of `mapset`, which is the name of @deckarep's package

Reply:
Not quite sure I understand what you mean! Would you mind expanding? @draychev